packages/mds-compliance-service/repository/entities/compliance-snapshot-entity.ts

```diff
@@ -38,15 +38,18 @@ export class ComplianceSnapshotEntity
   compliance_snapshot_id: ComplianceSnapshotEntityModel['compliance_snapshot_id']
 
   @Column('bigint', { transformer: BigintTransformer })
+  @Index()
```

Comment:
The absence of these `@Index()` decorators is why TypeORM was trying to delete them in the new migration. This was a previously existing issue.